install/debian-installation-guide.md

```diff
@@ -32,7 +32,7 @@
    ```bash
    $ ARCH=$(arch)
    $ source /etc/os-release
-   $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_${VERSION_ID}/ /' > /etc/apt/sources.list.d/kata-containers.list"
+   $ sudo sh -c "echo 'deb http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_${VERSION_ID:=9}/ /' > /etc/apt/sources.list.d/kata-containers.list"
    $ curl -sL  http://download.opensuse.org/repositories/home:/katacontainers:/releases:/${ARCH}:/master/Debian_${VERSION_ID}/Release.key | sudo apt-key add -
```

Comment:
Add the default here as well ? @yyyeerbo

Reply:
where? in curl -sL xxxx?
NO. VERSION_ID is set to 9 after sh -c executed, we don't have to set VERSION_ID again.

Reply:
This fixes the immediate problem, but is going to be fragile: we don't want to keep updating this doc as Debian moves it's stable release number. Looking at a Debian test box with Kata installed:

```sh
$ cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux buster/sid"
NAME="Debian GNU/Linux"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

That box *was* installed as Debian 9 (stretch). Looking at a vanilla stretch box:

```sh
$ cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 9 (stretch)"
NAME="Debian GNU/Linux"
VERSION_ID="9"
VERSION="9 (stretch)"
ID=debian
HOME_URL="https://www.debian.org/"
SUPPORT_URL="https://www.debian.org/support"
BUG_REPORT_URL="https://bugs.debian.org/"
```

A quick look shows the first system was subsequently `apt-get dist-upgrade`'d rather than `apt-get upgrade`'d. But this is still a bit surprising to me - the system *was* stretch - where did we say we wanted it to become sid?

Hence, I think we have a problem with the code in this doc which adds the unstable (sid) repo:

- https://github.com/kata-containers/documentation/blob/master/install/debian-installation-guide.md

We simply want to install a single package from sid and *never* install any other package from that version.